pages/api/feedback/index.js

```diff
@@ -0,0 +1,68 @@
+import { getSession } from 'next-auth/client'; // Session handling
+import { getPerformances } from '../performances/collect';
+import { getSchools } from '../schools/collect';
```

Comment:
Are these relative imports okay? @Anish-Agnihotri

Reply:
Think we should migrate to absolute, and if there isn't one for `@page` and that in `jsconfig.json`.